os/net/routing/rpl-classic/rpl-nbr-policy.h

```diff
@@ -0,0 +1,24 @@
+/*
+** ===================================================================
+** Project  : mermaid
+**
+** Copyright (c) 2020 Yanzi Networks. All Rights reserved. This
+** file is the confidential and proprietary property of Yanzi Networks
+** and the possession, use, duplication or transfer in any way of this
+** file requires a written agreement from Yanzi Networks.
+**
+**===================================================================
```

Comment:
These limitations might be a problem 😅

Reply:
Ahah very sorry, I forgot to update the header!
This and some more of our embedded code is green-lighted for open-sourcing.

Reply:
Fixed now!